src/extension.ts

```diff
@@ -97,11 +97,7 @@ export function activate(context: ExtensionContext) {
     }
 
     async function runSelectionInActiveTerm() {
-        const callableTerminal = await chooseTerminal(true);
-        if (callableTerminal === undefined) {
-            return;
-        }
-        runSelectionInTerm(callableTerminal, true);
+        window.showWarningMessage('This command has been removed. Enable setting "Always Use Active Terminal".');
```

Comment:
In my understanding, a warning is a message that complains but still works. Here do we still want the command to work? If it is not supposed to work any more, should we switch to an error instead?

Reply:
I thought about that too. We don't want the command to work. My logic was that an error makes it look like something has gone wrong, whereas this is more like an alert.

Reply:
Sounds fine.